CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [0.5.0]
+
+### Added
+
```

Comment:
i see this issue https://github.com/application-stacks/runtime-component-operator/issues/53 which has a label 0.5.0 is not added to this section here

Reply:
Thanks @yharish991. I updated it via PR https://github.com/application-stacks/runtime-component-operator/pull/86. Please review.